mne/viz/_figure.py

```diff
@@ -2204,7 +2229,11 @@ def _show_vline(self, xdata):
         else:
             self.mne.vline.set_xdata(xdata)
             self.mne.vline_hscroll.set_xdata(xdata)
-        self.mne.vline_text.set_text(f'{xdata:0.2f} s ')
+        if self.mne.show_real_time:
+            text = self._xtick_timestamp_formatter(xdata, None)
+        else:
+            text = f'{xdata:0.2f} s '
```

Comment:
I wonder if we should have a `start` parameter in `self._xtick_timestamp_formatter`, which would make this condition unnecessary? Also, what about the `None` argument (see my previous question)? Could you set the second argument to `None` by default?

Reply:
> I wonder if we should have a start parameter in self._xtick_timestamp_formatter, which would make this condition unnecessary?

This one I don't understand, what would the start-parameter do?

Reply:
I thought that we could always call `self._xtick_timestamp_formatter`, even for the default case where time starts at zero and is always shown in seconds. You could handle this case inside the method.

Reply:
I understand. This would probably also go nicely with the planned toggle-behaviour. I can look into it

Reply:
But in this specific case for the vline where an "s" is appended to the displayed time only for `time_format='float'`, I think it is still necessary to have this check inside `_show_vline`.